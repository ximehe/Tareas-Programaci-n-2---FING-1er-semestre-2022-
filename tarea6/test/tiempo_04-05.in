tiempo-abb 200000 1000 2
tiempo-cadena 600000 200000 2
tiempo-cola 200000 10 2
tiempo-colCadenas 1000000 2000000 2
tiempo-esAVL 400000 10 2
tiempo-filtradaOrdenada 500000 1 2
tiempo-iterador 100000 25000000 2
tiempo-map 100000 4 2
tiempo-menores 20 3 2
tiempo-menoresQueElResto 300000 10 2
tiempo-ordenadaPorModulo 30000 10 2
tiempo-palabras 8 200000 2
tiempo-pila 200000 10 2
Fin
