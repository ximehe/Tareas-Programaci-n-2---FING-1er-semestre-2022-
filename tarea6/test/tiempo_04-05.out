1>Bien.
2>Bien.
3>Bien.
4>Bien.
5>Bien.
6>Bien.
7>Bien.
8>Bien.
9>Bien.
10>Bien.
11>Bien.
12>Bien.
13>Bien.
14>Fin.
